src/LayerPanel/LayerPanelLayersPage/index.js

```diff
@@ -466,4 +464,4 @@ LayerPanelLayersPage.propTypes = {
   onLayerRemoved: PropTypes.func
 }
 
-export default LayerPanelLayersPage
\ No newline at end of file
+export default connectToMap(LayerPanelLayersPage)
```

Comment:
for some reason when i have connecttomap in this component it removes the layersicon when the layerpanel is opened...